frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts

```diff
@@ -116,9 +116,155 @@ describe('Pipelines', () => {
     cy.wait('@refreshVersions');
 
     // Verify the uploaded pipeline version is in the table
-    pipelinesTable.toggleExpandRowByIndex(0);
+    pipelinesTable.toggleExpandRowByName(initialMockPipeline.name);
     pipelinesTable.findRowByName('New pipeline version');
   });
+
+  it('delete single pipeline version', () => {
+    initIntercepts();
+    initPipelineIntercepts();
+    pipelinesGlobal.visit(projectName);
+    let deleted = false;
+    cy.intercept(
+      {
+        pathname: `/api/proxy/apis/v1beta1/pipeline_versions/${initialMockPipelineVersion.id}`,
+        method: 'POST',
+      },
+      (req) => {
+        deleted = true;
```

Comment:
I'm not sure if this is the best place to do this. The cypress intercept can be chained with a `then` I believe to update this variable right after the request is mocked, which I think is what is being aimed for here.

Reply:
Same comment applies to the other test you have.

Reply:
Let me give it a try.

Reply:
Yeah it works, I will update that.